shared/js/ui/templates/top-blocked-truncated-list-items.es6.js

```diff
@@ -18,11 +18,12 @@ module.exports = function (companyListMap) {
     })
 
     function getScssClass (companyName) {
-        var genericName = 'generic'
+        const genericName = 'generic'
+        const iconClassName = entityIconMapping[companyName]
 
         // TODO: remove Oath special case when we have an icon for it
-        if ((companyName !== 'oath') && majorTrackingNetworks[companyName]) {
-            return companyName
+        if ((companyName !== 'oath') && iconClassName) {
```

Comment:
Any reason we can't just remove this check since there's no longer a company with the name Oath?